### PR TITLE
D3M Polls december 13 2021

### PR DESCRIPTION
**Pull request by prose11**
Thanks for offering to take a look @astronautthis :)

### CODE REVIEW

File: governance/polls/DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [Maker Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, December 13 at 16:00 UTC.
+
+**This is an instant-runoff vote.** 
+- **You may rank any number of options that you would support in an executive vote.** 
+- **Unranked options signal that you would not support the option(s) in an executive vote.**
+- **If you have no preference to any of the listed options, you should vote 'Abstain' as your first and only choice.**
+
+## Review
+
+The community can vote in this poll to rank their support to recommend the following target [Debt Ceilings](https://makerdao.world/en/learn/governance/param-debt-ceiling) for the [DAI Direct Deposit Module (D3M)](https://mips.makerdao.com/mips/details/MIP50):
```
Comment (astronautthis) [inline governance/polls/DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021.md:28]: Template says MKR holders can vote, not community can vote.

File: governance/polls/DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [Maker Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, December 13 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to recommending the following rates spread between AAVEv2 borrow markets and Maker ETH-A [stability fee](https://makerdao.world/en/learn/governance/param-stability-fee) for the [DAI Direct Deposit Module (D3M)](https://mips.makerdao.com/mips/details/MIP50):
```
Comment (astronautthis) [inline governance/polls/DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021.md:27]: MKR holders can vote according to template

File: governance/polls/DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021.md (hunk 1 above)
Comment (prose11) [reply governance/polls/DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021.md]: Looks like our Generic templates need to be updated :)

File: governance/polls/DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021.md (hunk 2 above)
Comment (prose11) [reply governance/polls/DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021.md]: 👍

**Pull request merged**